modules/oc-mirror-migration-process.adoc

```diff
@@ -0,0 +1,154 @@
+
+.. For mirror-to-disk, run the following command:
++
+[source,terminal]
+----
+$ oc-mirror --config isc.yaml file://some-dir --v2
+----
+
+.. For disk-to-mirror, run the following command:
++
+[source,terminal]
+----
+$ oc-mirror --config isc.yaml --from file://some-dir docker://remote-registry --v2
+----
+
+.. For mirror-to-mirror, run the following command:
++
+[source,terminal]
+----
+$ oc-mirror --config isc.yaml --workspace file://some-dir docker://remote-registry --v2
+----
++
+[NOTE]
+====
+`--workspace` is now required for mirror-to-mirror operation.
+====
+
+. In v2, mirrored content is stored in different locations. You must check the cluster-resources directory inside their working directory for IDMS, ITMS, `CatalogSource`, and `ClusterCatalog` using the following command :
++
+[source,terminal]
+----
+$ ls <workspace>/cluster-resources/
+----
+
+. [*Optional*] oc-mirror plugin v2 replaces ICSP with IDMS/ITMS. Before applying the new configuration, delete the outdated ICSP entries to avoid conflicts using the following command:
++
+[source,terminal]
+----
+$ oc delete <ICSP/IDMS/ITMS> <1>
+----
+<1> Specify the `ICSP/IDMS/ITMS` name.
+
+. To remove images mirrored using oc-mirror plugin v1, follow these steps:
++
+[IMPORTANT]
+====
+oc-mirror plugin v2 does not automatically prune the destination registry, unlike oc-mirror plugin v1. If you want to clean up images no longer needed, use the delete functionality in v2 with `--delete-v1-images`.
+
+Once all images mirrored with oc-mirror plugin v1 are removed, you no longer need to use this flag. If you need to delete images mirrored with oc-mirror plugin v2, do not set `--delete-v1-images`.
+====
+
+.. Mirror the image using oc-mirror plugin v1.
+
+.. Upgrade the `ImageSetConfiguration` API version from `v1alpha2` (v1) to `v2alpha1` (v2), then run the following command:
++
+[source,terminal]
+----
+$ oc-mirror -c isc.yaml file://some-dir --v2
+----
+
+.. Generate a delete manifest and delete v1 images using the following command:
++
+[source,terminal]
+----
+$ oc-mirror delete --config=delete-isc.yaml --generate --delete-v1-images --workspace file://some-dir docker://remote-registry --v2
+----
+
+.. Delete images based on the generated manifest:
++
+[source,terminal]
+----
+$ oc-mirror delete --delete-yaml-file some-dir/working-dir/delete/delete-images.yaml docker://remote-registry --v2
+----
+
+.Verification
+
+. Once the migration is complete, verify that mirrored images and catalogs are available. Ensure:
+
+- No errors or warnings occurred during mirroring.
+- No error file was generated (`working-dir/logs/mirroring_errors_YYYYMMdd_HHmmss.txt`).
+
+. Verify that mirrored images and catalogs are available using the following the commands:
++
+[source,terminal]
+----
+$ oc get catalogsource -n openshift-marketplace
+$ oc get imagedigestmirrorset,imagetagmirrorset -n openshift-config
+----
+For more information, refer to _mirroring in a disconnected environment_.
```

Comment:
we don't tell them to apply these to the cluster first...
so if they follow this migration guide to the letter, catalogsource hasn' t been applied yet...
I would rather we point them back to the v2 documentation (Configuring your cluster to use the resources generated by oc-mirror plugin v2) instead